src/main/scala/org/constellation/primitives/Genesis.scala

```diff
@@ -164,23 +164,17 @@ object Genesis extends StrictLogging {
 
     dao.metrics.updateMetric("genesisHash", go.genesis.soeHash)
 
-    GenesisObservationWriter
-      .writeToDisk(go, dao.genesisObservationPath)(IO.contextShift(ConstellationExecutionContext.bounded))
-      .flatTap {
-        case Success(value)     => Logger[IO].info(s"Genesis observation saved successfully ${value.pathAsString}")
-        case Failure(exception) => Logger[IO].error(s"Cannot save genesis observation to disk ${exception.getMessage}")
+    //TODO: `unsafeRunSync` temporary solution before refactoring Genesis
+    dao.genesisObservationWriter
+      .write(go)
+      .value
+      .map {
+        case Left(value) => logger.error(s"Cannot write genesis observation ${value.exceptionMessage}")
+        case Right(_)    => logger.info("Genesis observation saved successfully")
       }
```

Comment:
Eventually more "standardized" and "bookish" way of calling function when Left or when Right by using `fold`:

```scala
.fold(
   value => logger.error(s"Cannot write genesis observation ${value.exceptionMessage}"),
   _ => logger.info("Genesis observation saved successfully")
)
```